docs/en/getting_started.md

```diff
@@ -92,6 +92,8 @@ A small test dataset for quick demo can be downloaded [here](https://openxrlab-s
 
 For optimization-based approaches, it does not require any pretrained model. Taking [MVPose](../../configs/mvpose/) as an example, it can be run on Shelf_50 as
 
+For optimization-based approaches, it does not require any pretrained model. Taking [MVPose](https://zju3dv.github.io/mvpose/) as an example, it can be run as
```

Comment:
`it does not require any pretrained model`, but we do have a pretrained reid model, don't we?

Reply:
We need a pretrained reid model.